spec/features/families/admin_views_all_families_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "Admin can see all the families when in families#index" do
+  describe "when click on the 'All' button" do
+    scenario "successfully", js: true do
+      create(:all_true_family)
+      create(:all_false_family)
+
+      visit families_path
+
+      expect(page).to have_text(I18n.t("families.index.buttons.all"))
+      expect(page).to have_text("false")
+      expect(page).to have_text("true")
```

Comment:
Actually above comments apply to this spec

Reply:
Done in sha `f995a5f`